libbeat/docs/upgrading.asciidoc

```diff
@@ -93,6 +77,45 @@ we've provided a migration tool to help you migrate your configurations. For
 more information, see the
 https://www.elastic.co/blog/beats-6-7-0-released[Beats 6.7.0 release blog].
 
+==== Upgrade {beats} binaries to 7.0
+
+// REVIEWERS: I mirrored the steps for Elasticsearch because I figured they
+// would be similar, but I have not tried these steps on all platforms. Please
+// confirm that they are accurate.
+
+Before upgrading:
+
+. Back up the `data` and `config` directories. 
```

Comment:
How should people do this? If it's just by copying these files somewhere else, maybe we could state that:

> Back up the `data` and `config` directories by copying them to another location.

I think it's worth being clear on this point because we have the Snapshots API which we refer to when advising the user to back up their ES data.

Reply:
Excellent point. Thank you.